### PR TITLE
Optionally hide percentile information display in the web UI.

### PR DESCRIPTION
**Pull request by wtritch**

### CODE REVIEW

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/configuration/TenantProperties.java [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
     private boolean transferAccessEnabled;
     private String translationLocation;
+    private boolean enablePercentileDisplay;
```
Comment (wtritch) [inline common/src/main/java/org/opentestsystem/rdw/reporting/common/configuration/TenantProperties.java]: Just added it to existing tenant properties.

File: reporting-service/src/main/java/org/opentestsystem/rdw/reporting/user/AppSettings.java [hunk 2]
```diff
@@ -76,4 +76,10 @@ public boolean isTransferAccess() {
         return tenantProperties.isTransferAccessEnabled();
     }
 
+    /**
+     * @return True if UI should display norms percentile information
+     */
+    public boolean isEnablePercentileDisplay() {
+        return tenantProperties.isEnablePercentileDisplay();
+    }
```
Comment (wtritch) [inline reporting-service/src/main/java/org/opentestsystem/rdw/reporting/user/AppSettings.java]: this is piping it through to the "user" payload returned to the front-end via the "/user" endpoint.

File: webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.html [hunk 3]
```diff
@@ -157,7 +157,7 @@
                 <i class="fa fa-book"></i> {{'labels.groups.results.assessment.instruct' | translate }}
               </button>
             </li>
-            <li *ngIf="assessmentExam.assessment.isIab">
+            <li *ngIf="enablePercentileDisplay && assessmentExam.assessment.isIab">
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.html]: Just added it to the display *ngIf

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/configuration/TenantProperties.java [hunk 4]
```diff
@@ -39,4 +40,18 @@ public String getTranslationLocation() {
...
         this.translationLocation = translationLocation;
     }
+
+    /**
+     * This property controls whether or not to display the norms percentile
+     * information button within the web UI.
+     *
+     * @return True if the UI should display norms percentile information
+     */
+    public boolean isEnablePercentileDisplay() {
```
Comment (esotrope) [inline common/src/main/java/org/opentestsystem/rdw/reporting/common/configuration/TenantProperties.java]: would read better as percentileDisplayEnabled (isPercentileDisplayEnabled)

Comment (wtritch) [reply common/src/main/java/org/opentestsystem/rdw/reporting/common/configuration/TenantProperties.java]: ...I knew there was going to be a naming change, it it's going to touch every single file.  @malaffoon any other names come to mind?  I don't mind renaming once...

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/configuration/TenantProperties.java (hunk 1 above)
Comment (esotrope) [reply common/src/main/java/org/opentestsystem/rdw/reporting/common/configuration/TenantProperties.java]: should we have a default value in the yml?

Comment (wtritch) [reply common/src/main/java/org/opentestsystem/rdw/reporting/common/configuration/TenantProperties.java]: I've been debating that for all of our properties... but I think we decided to not include every configurable property in the .yml files... I don't know if this one and/or translation location and/or transfer access and/or etc should be included.  We may want to sit down and consider them all one-by-one?

File: reporting-service/src/main/java/org/opentestsystem/rdw/reporting/user/AppSettings.java (hunk 2 above)
Comment (esotrope) [reply reporting-service/src/main/java/org/opentestsystem/rdw/reporting/user/AppSettings.java]: hm as the props grow in number i feel like we might benefit from being explicitly listing them all in the prop files upfront

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/configuration/TenantProperties.java (hunk 4 above)
Comment (wtritch) [reply common/src/main/java/org/opentestsystem/rdw/reporting/common/configuration/TenantProperties.java]: done

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/configuration/TenantProperties.java (hunk 1 above)
Comment (malaffoon) [reply common/src/main/java/org/opentestsystem/rdw/reporting/common/configuration/TenantProperties.java]: My vote is to use the annotated config files in the RDW project. But yeah, i get that there is some distance to that project.

**Pull request merged**